deploy/containers/credhub_setup/pkg/quarks/deployment.go

```diff
@@ -0,0 +1,20 @@
+package quarks
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+)
+
+// GetDeploymentName returns the deployment name, which is embedded in some
+// names and paths.
+func GetDeploymentName(ctx context.Context) (string, error) {
+	mountRoot := GetMountRootFromContext(ctx)
+	deploymentFileName := filepath.Join(mountRoot, "run", "pod-info", "deployment-name")
```

Comment:
But that would be weird, since nobody else should have to know that the deployment name comes from a file (as opposed to, say, an environment variable).  I could see it being a `const`, but that's… kind of unnecessary?